cmd/main.go

```diff
@@ -135,7 +138,7 @@ func initializeServerPairs(server *server.Server, services *Services, repositori
 			services.messages))
 
 	server.AddPair(ethereum.NewWatcher(services.contracts, clients.Ethereum, configuration.Validator.Clients.Ethereum),
```

Comment:
I think that we should think about extending the watcher/handler pairs so that we have: `1 -> *` relation and not `1 -> 1`.

@failfmi what do you think?